ionic2/app/pages/about/about.page.html

```diff
@@ -24,16 +24,12 @@
     intelligently guess when and where our Pokemons will pop up.
   </p>
   <ion-title>Our Team</ion-title>
-  <h5>Pokémon Trainer</h5>
-  <a>Christian Dallago</a><br><a>Dr. Guy Yachdav</a><br><a>Dr. Tatyana Goldberg</a><br><a>Juan Miguel Cejuela</a><br>
-  <h5>Pokémon Data Analysts</h5>
-  <p>Annette Köhler<br>Fabian Buske<br>Jonas Heintzenberg<br>Samit Vaidya<br>Swathi S Sunder<br>Vivek Sethia</p>
-  <h5>Pokémon Clairvoyants</h5>
-  <p>Aurel Roci<br>Benjamin Strobel<br>Marcel Wagenländer<br>Matthias Baur<br>Siamion Karcheuski<br>Timur Khodzhaev</p>
-  <h5>Pokémon Whisperer</h5>
-  <p>Amr Abdelraouf<br>Hannes Dorfmann<br>Karen Reyna<br>Philipp Dowling</p>
-  <h5>Pokémon World Cartographers</h5>
-  <p>Elma Gazetic<br>Faris Cakaric<br>Gani Qinami<br>Oleksandr Fedotov<br>Paul Gualotuna<br>Timo Ludwig</p>
-  <h5>Pokémon Catcher</h5>
-  <p>Alexander Lill<br>Georgi Aylov<br>Gilles Tanson<br>Jochen Hartl<br>Josef Brandl<br>Mustafa Kaptan<br>Philippe Buschmann<br>Wolfgang Hobmaier</p>
+  <ion-list no-lines *ngFor="let group of people">
+    <ion-list-header>
+      {{group.alias}}
+    </ion-list-header>
+    <ion-item *ngFor="let person of group.member">
+      <a href="{{person.link}}">{{person.name}}</a>
```

Comment:
The angular 2 way of doing this would be `[href]="person.link"`. The reason for this is that until angular compiles the HTML the `{{person.link}}` is not translated. So some browser could throw warnings that `{{person.link}}` is not a valid URL. Even worse, when angular takes a long time to load (like on older devices) the user could click on the link before it has been translated.